net/host.go

```diff
@@ -177,7 +177,6 @@ func (h *host) Start() error {
 	h.h.SetStreamHandler(protocol.ID(h.protocolID+queryID), h.handleQueryStream)
 	h.h.SetStreamHandler(protocol.ID(h.protocolID+swapID), h.handleProtocolStream)
 
-	h.h.Network().SetConnHandler(h.handleConn)
```

Comment:
`SetConnHandler` was removed: https://github.com/libp2p/go-libp2p-core/pull/217
We were only using it to log a message, so I just removed it from our code too.